packages/hint/docs/user-guide/concepts/parsers.md

```diff
@@ -16,40 +16,35 @@ configuration file:
 
 ## List of official `parser`s
 
```

Comment:
Did you want to include `babel-config` here? (I did in my PR):
```suggestion

* [`babel-config`][@hint/parser-babel-config] A `Babel configuration` parser
  which validates the provided `json` so hints analyzing `.babelrc` files can
  be built.

```

Reply:
Done